tests/integration/test_parallel_replicas_no_replicas/test.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+from helpers.client import QueryRuntimeException
+
+cluster = ClickHouseCluster(__file__)
+
+initiator = cluster.add_instance(
+    "initiator", main_configs=["configs/remote_servers.xml"], with_zookeeper=True
+)
+
+
+@pytest.fixture(scope="module")
+def start_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def create_tables(cluster, table_name):
+    initiator.query(f"DROP TABLE IF EXISTS {table_name} SYNC")
+
+    initiator.query(
+        f"CREATE TABLE IF NOT EXISTS {table_name} (key Int64, value String) Engine=ReplicatedMergeTree('/test_parallel_replicas/shard1/{table_name}', 'r1') ORDER BY (key)"
+    )
+
+    # populate data
+    initiator.query(
+        f"INSERT INTO {table_name} SELECT number % 4, number FROM numbers(1000)"
+    )
+
+
+@pytest.mark.parametrize("skip_unavailable_shards", [1, 0])
+def test_skip_all_replicas(start_cluster, skip_unavailable_shards):
+    cluster_name = "test_1_shard_3_unavaliable_replicas"
+    table_name = "tt"
+    create_tables(cluster_name, table_name)
+
+    with pytest.raises(QueryRuntimeException):
```

Comment:
maybe makes sense to check for specific error type, to make sure it is not smth irrelevant like 'unknown cluster' or whatever else